src/components/container/InboxItemList/InboxItemList.tsx

```diff
@@ -0,0 +1,130 @@
+import './InboxItemList.css';
+import MyDataHelps, { InboxItem, InboxItemQueryParameters, InboxItemSortColumn, InboxItemStatus, InboxItemType, InboxMessage, InboxResource, InboxSurvey, SortOrder } from '@careevolution/mydatahelps-js';
+import { InboxMessageListItem, InboxResourceImageAlignment, InboxResourceListItem, InboxSurveyListItem, InboxSurveyVariant, LoadingIndicator } from '../../presentational';
+import { useInitializeView } from '../../../helpers/Initialization';
+import inboxDataService from '../../../helpers/Inbox/inbox-data';
+import { InboxItemListCoordinatorContext } from '../InboxItemListCoordinator';
+import { viewInboxMessage } from '../../view/InboxMessageView';
+import { InboxItemListPreviewState, previewData } from './InboxItemList.previewData';
+
+export interface InboxItemListProps {
+    previewState?: InboxItemListPreviewState;
+    title?: string;
+    type?: InboxItemType | InboxItemType[];
+    status?: InboxItemStatus | InboxItemStatus[];
+    category?: string;
+    sortOrder?: SortOrder;
+    sortColumn?: InboxItemSortColumn;
+    endsAfter?: string;
+    endsBefore?: string;
+    limit?: number;
+    messageViewerUrl?: string;
+    emptyText?: string;
+    showTitleWhenEmpty?: boolean;
+    surveyVariant?: InboxSurveyVariant;
+    resourceImageAlignment?: InboxResourceImageAlignment;
+    hideLoadingIndicator?: boolean;
+    onItemsLoaded?: (items: InboxItem[]) => void;
+    syncOnChanges?: boolean;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: InboxItemListProps) {
+    const [loading, setLoading] = useState<boolean>(true);
+    const [items, setItems] = useState<InboxItem[]>();
+
+    const onItemsLoaded = (items: InboxItem[]): void => {
+        setItems(items);
+        if (props.onItemsLoaded) {
+            props.onItemsLoaded(items);
+        }
+    };
+
+    const loadData = (): void => {
+        setLoading(true);
+
+        if (props.previewState) {
+            onItemsLoaded(previewData[props.previewState].items);
+            setLoading(false);
+            return;
+        }
+
+        let query: InboxItemQueryParameters = {
+            type: props.type,
+            status: props.status,
+            category: props.category,
+            sortOrder: props.sortOrder,
+            sortColumn: props.sortColumn,
+            endsAfter: props.endsAfter,
+            endsBefore: props.endsBefore
+        };
+        inboxDataService.loadInboxItems(query).then(items => {
+            onItemsLoaded(items);
+            setLoading(false);
+        });
+    };
+
+    let inboxContext = useContext(InboxItemListCoordinatorContext);
+    if (inboxContext && props.syncOnChanges) {
+        inboxContext.registerChangeListener(loadData);
+    }
+
+    useInitializeView(loadData, [], [props.previewState]);
+
+    if (loading && !items) {
+        return <div className="mdhui-inbox-item-list" ref={props.innerRef}>
+            {props.title && props.showTitleWhenEmpty &&
```

Comment:
Nothing wrong with the code here but more the presentation. 
When I pull up the Inbox View the font is very similar to the survey description. It took a moment to orient my eyes with what I was looking at vs being clearly drawn to Message/Survey/Resource/

Reply:
Would you be able to provide a screenshot?

I think the Inbox UI is likely to undergo some more polish and improvement in the very near future as we get some mileage on it in the wild, so I'd rather not spend too much time trying to make it "perfect" without some user feedback, but if there is an obvious way we can improve things out of the box, I'm open to that.

Reply:
@greinard 
![image](https://github.com/CareEvolution/MyDataHelpsUI/assets/25618674/4005ecbc-7bce-462a-94c0-8de451cb7886)

Reply:
Some notes from slack convo. Noting here for posterity
- Font size of message title vs task title would look nicer if consistent
- The view has not limit to how many messages/surveys/etc. Would reconsider showing message titles in bold as it could be a list of bold text
- Section titles are like floating signs. My eye goes toward the bold message and survey titles, then the section Message/Surveys/Resources come into focus. Does not look as clean as it could

Reply:
As discussed, this design is likely to have some iterations in the near future to adjust for user feedback.  Since this design has been used previously without issue, I hesitate to modify it too much until we get some real world feedback.